src/views/Conversion.vue

```diff
@@ -222,15 +223,20 @@ export default class Conversion extends Mixins(
     const items = await Promise.all([
       this.fetchBusinessInfo(),
       AuthServices.fetchAuthInfo(this.getBusinessId),
-      this.fetchAddresses(),
       this.fetchOrgPersons(OrgPersonTypes.PARTIES)
     ])
 
-    if (items.length !== 4) throw new Error('Failed to fetch entity snapshot')
+    if (items.length !== 3) throw new Error('Failed to fetch entity snapshot')
+
+    const addresses = await this.fetchAddresses()
+      .catch(reason => {
+        if (reason.response?.status === NOT_FOUND) return { businessOffice: null }
```

Comment:
Need to also check message here. Maybe Lewis will submit this change and I can remove it from this PR.

Reply:
Lewis will add the message string check in a future PR.